libkbfs/folder_update_prepper.go

```diff
@@ -331,6 +331,13 @@ type pathTreeNode struct {
 	mergedPath path
 }
 
+type prepFolderFlags byte
+
+const (
+	// The blocks of indirect files should be copied.
+	prepFolderCopyIndirectFileBlocks prepFolderFlags = 1 << iota
```

Comment:
Does this actually need the bitpacking, though? Why not different types for the different flags, like:

```
type prepFolderCopyBehavior int

const (
  prepFolderCopyIndirectFileBlocks prepFolderCopyBehavior = 1
  prepFolderDontCopyIndirectFileBlocks ...
)

type prepFolderFooBehavior int
```

It's more verbose, but perhaps less error-prone. What do you think?